base_agent/sql_memory.py

```diff
@@ -1154,3 +1162,19 @@ def safe_unpickle(self, bs):
                     delattr(obj, "__had_attr_" + attr)
                     setattr(obj, attr, self._safe_pickle_saved_attrs[obj.pickled_attrs_id][attr])
         return obj
+
+
+if __name__ == "__main__":
+
+    class P:
```

Comment:
Is this being used somewhere ? or was it for local testing ? :) If the latter, we should remove